bugsnag-android-core/src/main/java/com/bugsnag/android/NativeStackframe.kt

```diff
@@ -0,0 +1,52 @@
+package com.bugsnag.android
+
+import java.io.IOException
+
+/**
+ * Represents a single native stackframe
+ */
+class NativeStackframe internal constructor(
```

Comment:
```suggestion
internal class NativeStackframe internal constructor(
```

Confusingly the default visibility in Kotlin [is public](origin/5317-conditional-ndk-anr-2). As there isn't any requirement to expose `NativeStackframe` as part of public API it can be hidden for now.

Reply:
This also needs to be accessible to AnrPlugin. How can I keep it "internal" but accessible to that plugin?

Reply:
This one I couldn't set to internal because AnrPlugin needs to use it directly...